### DISCUSSION TITLE
Auto-Southwest Check-In v6.1

### DISCUSSION THREAD

**Discussion by jdholtz** [jdholtz/auto-southwest-check-in#163] [Announcements]
**Concise Changelog**: [CHANGELOG.md](https://github.com/jdholtz/auto-southwest-check-in/blob/master/CHANGELOG.md#61-2023-09-28)

**Full Changelog**: https://github.com/jdholtz/auto-southwest-check-in/compare/v6.0...v6.1

<hr /><em>This discussion was created from the release <a href='https://github.com/jdholtz/auto-southwest-check-in/releases/tag/v6.1'>Auto-Southwest Check-In v6.1</a>.</em>

**Comment by wrusry** [comment]
Hello, I had the failure in v 6.0 of too many requests on my outbound checkin. So, I upgraded to version 6.1 to try to use it to check in for my return flight on my computer which doesn't go to sleep and has a UPS to it. It also failed for "too many requests". See the below image. Previously, I had used southwest.py successfully, but I had the issue of matching my chrome with your version, so in my last successful usage, I had to downgrade my Chrome browser in order to make it work. I was excited to see that the program was no longer dependent on the Chrome version. Please let me know if you need any more info from my for troubleshooting purposes. 

![image](https://github.com/jdholtz/auto-southwest-check-in/assets/44855611/1a176650-fd1d-4d70-bd34-05264f87bf0c)

**Reply by jdholtz** [reply]
Discussed in #172
